schema/types/isolator.go

```diff
@@ -69,8 +112,17 @@ func (is *Isolators) Unrecognized() Isolators {
 // serialized as any arbitrary JSON blob. Specific Isolator types should
 // implement this interface to facilitate unmarshalling and validation.
 type IsolatorValue interface {
+	// UnmarshalJSON unserialize a JSON-encoded isolator
 	UnmarshalJSON(b []byte) error
+	// AssertValid returns a non-nil error value if an IsolatorValue is not valid
+	// according to appc spec
 	AssertValid() error
+	// MaxInstances specifies the maximum number of allowed isolator instances
+	// for this isolator type
+	MaxInstances() uint
```

Comment:
> (aka math.MaxUint32?)

No, value depends on arch-specific uint size.

> I don't see a use case for other values. Could it be multipleAllowed() bool instead?

To be honest I don't have a usecase to propose, but I (generally) prefer to sketch those interfaces with an open/close approach. No strong opinions here, I can rework it if you find it prettier. Thoughts?